docs/consume-packages/Finding-and-Choosing-Packages.md

```diff
@@ -13,19 +13,31 @@ When starting any .NET project, or whenever you identify a functional need for y
 
 ## Finding packages
 
-When you visit nuget.org or open the Package Manager UI in Visual Studio, you see a list of packages sorted by total downloads. This immediately shows you the most widely-used packages across the millions of .NET projects. There's a good chance, then, that at least some of the packages listed on the first few pages will be useful in your projects.
+When you visit nuget.org or open the Package Manager UI in Visual Studio, you see a list of packages sorted by relevancy. This shows you the most widely used packages across all .NET projects. There's a good chance that some of these packages may be useful for your own projects!
 
 ![Default view of nuget.org/packages showing the most popular packages](media/Finding-01-Popularity.png)
 
-Notice the **Include prerelease** option on the upper right of the page. When selected, nuget.org shows all versions of packages including beta and other early releases. To show only stable released, clear the option.
-
-For specific needs, searching by tags (within the Visual Studio Package Manager or on a portal like nuget.org) is the most common means of discovering a suitable package. For example, searching on "json" lists all NuGet packages that are tagged with that keyword and thus have some relationship to the JSON data format.
+Notice the **Filter** button on the upper right of the page. When clicked, the Advanced Search panel expands to present sorting and filtering options.
 
 ![Search results for 'json' on nuget.org](media/Finding-02-SearchResults.png)
 
-You can also search using the package ID, if you know it. See [Search Syntax](#search-syntax) below.
+You can use the **Package type** filter to only show packages of a specific type:
+- **`All types`**: This is the default behavior. It shows all packages regarding of their type.
+- **`Dependency`**: This only shows NuGet dependency packages which represent libraries.
```

Comment:
I'm not sure the libraries distinction is all that clear to our customers. What do you think of something like this?

```suggestion
- **`Dependency`**: Regular NuGet packages that can be installed into your project.
```